interface/documents/upload.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+* Drag and Drop file uploader.
+*
+* Copyright (C) 2017 Sherwin Gaddis sherwingaddis@gmail.com
```

Comment:
We are trying to standardize our docblocks. Examples here http://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly#Examples_for_OpenEMR